### PR TITLE
Adds a hotkey that puts you in throw mode aslong as you hold it

### PR DESCRIPTION
**Pull request by ghost**
🆑 Qustinnus
add: Adds a hotkey that puts you in throw mode aslong as you hold it. Defaults to Space
/🆑

I personally feel like a very active action like throwing makes more sense as a held button, as I often accidentaly yeet something im holding as I toggled throw mode before, or want to quickly switch between catching / fighting.

The toggle still works the same, pressing R will turn on throw or clicking the button will turn it on, but the hold takes "priority" over the toggle, (So pressing r or the button while holding space or even throwing something wont turn it off, you need to let go of space)

You can completely ignore this option if its not your cup of tea. I made the default bind space as its easy to reach when using wasd to move.

### CODE REVIEW

File: code/datums/keybinding/carbon.dm [hunk 1]
```diff
@@ -21,6 +21,27 @@
...
+	description = "Hold this to turn on throw mode, and release it to turn off throw mode"
+	category = CATEGORY_CARBON
+	keybind_signal = COMSIG_KB_CARBON_HOLDTHROWMODE_DOWN
+
+/datum/keybinding/carbon/hold_throw_mode/down(client/user)
+	. = ..()
+	if(.)
+		return
+	var/mob/living/carbon/C = user.mob
+	C.throw_mode_on(THROW_MODE_HOLD)
```
Comment (Mothblocks) [inline code/datums/keybinding/carbon.dm:37]:
```suggestion
	var/mob/living/carbon/carbon_user = user.mob
	carbon_user.throw_mode_on(THROW_MODE_HOLD)
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]
so close

**Comment by Mothblocks** [comment]
undraft if ready

**Review by Mothblocks — APPROVED** [review]
It's good, though I'd like to see this alongside a custom cursor, as otherwise it's not obvious when you press space and get no feedback, especially as a new player who likely hasn't sifted through the countless keybinds. Waiting to see your thoughts.

**Comment by Qustinnus** [comment]
cant find a spriter

**Comment by Mothblocks** [comment]
Shame, it's not necessary at this point in time but something that really needs to get done.

**Pull request merged**